src/workspacer.FocusIndicator/FocusIndicatorPlugin.cs

```diff
@@ -7,8 +7,6 @@ public class FocusIndicatorPlugin : IPlugin
 
         private FocusIndicatorForm _form;
 
-        public FocusIndicatorPlugin() : this(new FocusIndicatorPluginConfig()) { }
-
```

Comment:
Any reason to remove this constructor? Someone might be depending on it.

Reply:
No reason other than it not being used internally, and that the config extensions would usually be used for the configuration over the constructor. It has been added back in.